### PR TITLE
Fix: Section type constants are shown as integers #697

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by arsenius7** [comment]
LGTM

File: requirements_dev.txt [hunk 1]
```diff
@@ -29,6 +29,7 @@
 mock>=1.3.0
 pylint==1.7.4
 pycodestyle==2.3.1
+pandas==0.22.0
```
Comment (mgeplf) [inline requirements_dev.txt]: adding pandas as a dependency just to run a test seems like overkill

Comment (wizmer) [reply requirements_dev.txt]: I know but it's a only dev dependency so I don't think it's a big deal.
And that's the only way I can think of to test the expected behaviour

Comment (arsenius7) [reply requirements_dev.txt]: If it is such a big deal, checking the type of the value might suffice.

Comment (wizmer) [reply requirements_dev.txt]: Ah wait, maybe I can check on a np.array instead. Let me try

**Pull request merged**